content/quickstart/kubernetes/introduction.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Introduction
+menu:
+  quickstart:
+    parent: kubernetes
+    identifier: kubernetes-using-pulumi
+    weight: 1
+---
+
+## Using Pulumi
+
+Pulumi can be used with Kubernetes in many ways. Users can:
+
+-   Provision a managed Kubernetes cluster on a cloud provider.
+-   Provision the IaaS resources for a self-managed Kubernetes cluster.
+-   Create Kubernetes API resources and deploy app workloads to any existing Kubernetes cluster (managed or
+    self-managed) with only a `kubeconfig` required.
+
+If you have already provisioned a Kubernetes cluster and have set up `kubectl`
+to connect to it, the Pulumi CLI should "just work" transparently out of the box
+after importing the [`pulumi/kubernetes`][pulumi-k8s] package into your program.
+
+Pulumi supports any Kubernetes version that is currently supported by 
+Kubernetes [upstream][upstream].
+
+## Why Pulumi?
+
+In addition to being compatible with all existing Kubernetes workflows, Pulumi provides several
+advantages:
```

Comment:
Probably worth linking to https://blog.pulumi.com/pulumi-a-better-way-to-kubernetes